bin/main.rkt

```diff
@@ -538,4 +544,4 @@
                                   (error "Yosys failed."))))
        (output-port))]
 
-     [_ (error "Invalid output format.")])])
+     [_ (error "Invalid output format.")])])
```

Comment:
can you undo this change? i'm ot sure why your editor keeps making it

Reply:
> can you undo this change? i'm ot sure why your editor keeps making it

Yeah I can do undo it. I think this happened after I ran `raco fmt` on the file.